Content.Shared/PlantAnalyzer/PlantAnalyzerScannedSeedPlantInformation.cs

```diff
@@ -0,0 +1,94 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.PlantAnalyzer;
+
+/// <summary>
+///     The information about the last scanned plant/seed is stored here
+/// </summary>
+[Serializable, NetSerializable]
+public sealed class PlantAnalyzerScannedSeedPlantInformation : BoundUserInterfaceMessage
+{
+    public readonly NetEntity? TargetEntity;
+
+    public string SeedName = "";
+    public string SeedYield = "";
+    public string SeedPotency = "";
+    public string Repeat = "";
+    public string SeedChem = "";
+    public string SeedMutations = "";
+    public string ExudeGases = "";
+    public Boolean IsTray;
+    public Boolean ScanMode;
+
+    public string NutrientConsumption = "";
+    public string WaterConsumption = "";
+    public string IdealHeat = "";
+    public string HeatTolerance = "";
+    public string IdealLight = "";
+    public string LightTolerance = "";
+    public string ToxinsTolerance = "";
+    public string LowPresssureTolerance = "";
+    public string HighPressureTolerance = "";
+    public string PestTolerance = "";
+    public string WeedTolerance = "";
+
+    public string Lifespan = "";
+    public string Maturation = "";
+    public string GrowthStages = "";
+    public string PlantSpeciation = "";
+
+    public string Tolerances = "";
+    public string Endurance = "";
+
+    public PlantAnalyzerScannedSeedPlantInformation(NetEntity? targetEntit, string seedName, string seedChem, string plantHarvestType,
+        string exudeGases, string seedMutations, Boolean isTray, string plantSpeciation, string tolerances, string generalTraits, Boolean scanMode)
+    {
+        TargetEntity = targetEntit;
+        ScanMode = scanMode;
+
+        SeedName = seedName;
+        SeedChem = seedChem;
+        Repeat = plantHarvestType;
+        ExudeGases = exudeGases;
+
+        IsTray = isTray;
+
+        SeedMutations = seedMutations;
+        PlantSpeciation = plantSpeciation;
+
+        String[] arrGeneral = generalTraits.Split(";");
+        //SeedName = arrGeneral[0];
+        Endurance = arrGeneral[1];
+        SeedYield = arrGeneral[2];
+        Lifespan = arrGeneral[3];
+        Maturation = arrGeneral[4];
+        GrowthStages = arrGeneral[5];
+        SeedPotency = arrGeneral[6];
+
+        if (scanMode || SeedName.Contains("NanoTrasen")) //advanced scan mode OR its a roundstart prototype seed
```

Comment:
Why should round-start prototypes be exempted?

Reply:
My logic was that those seeds are provided by NT so we should know the base stats, making it more obvious how we mutated a plant without having to screenshot a starting scan for every plant.
Which I still like, but my implementation was bad (human-readable string as condition....) so I removed it.

It does still show those prototype seeds with a distinct name in the scanner but only for immersion and no mechanical influence.